cscs-checks/cuda/cuda_checks.py

```diff
@@ -12,7 +12,6 @@ def __init__(self):
         if self.current_system.name == 'kesch':
```

Comment:
This set of tests still needs to be ported to Tsa/Arolla, or are they discarded?